catroid/src/main/java/org/catrobat/catroid/content/bricks/ForItemInUserListBrick.java

```diff
@@ -36,7 +36,6 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.UUID;
 
 public class ForItemInUserListBrick extends UserDataBrick implements CompositeBrick {
 
```

Comment:
Just exchange the following line with my suggestion. 

```
private transient EndBrick endBrick = new EndBrick(this,R.layout.brick_loop_end);
```
The second parameter is actually only needed for the classes that use R.layout.brick_if_end_if
You also have to change it for the clone method. 


And you can delete all the stuff on the bottom.